### PR TITLE
Add option to pass autoCorrect prop to the text input component

### PR DESCRIPTION
**Pull request by KristineTrona**
Added ability to disable autoCorrect on the text input by passing an ``autoCorrect`` prop.

### CODE REVIEW

File: src/CNTextInput.js [hunk 1]
```diff
@@ -1306,7 +1306,7 @@ class CNTextInput extends Component {
 
     render() {
       const {
-        items, foreColor, style, returnKeyType, styleList
+        items, foreColor, style, returnKeyType, styleList, autoCorrect
```
Comment (pacozaa) [inline src/CNTextInput.js]: `const { items, foreColor, style, returnKeyType, styleList, ...restProps} = this.props`

File: src/CNTextInput.js [hunk 2]
```diff
@@ -1336,7 +1336,8 @@ class CNTextInput extends Component {
           onFocus={this.onFocus}
           onBlur={this.onBlur}
           onContentSizeChange={this.handleContentSizeChange}
-		      placeholder={this.props.placeholder}
+          placeholder={this.props.placeholder}
+          autoCorrect={autoCorrect}
```
Comment (pacozaa) [inline src/CNTextInput.js]: then `autoCorrect={restProps.autoCorrect}`

**Comment by KristineTrona** [comment]
@pacozaa Thank you for the feedback. Please see the changes.

**Comment by pacozaa** [comment]
@imnapo @KristineTrona You guys are awesome! Keep up good work!

**Pull request merged**